README.md

```diff
@@ -220,6 +221,39 @@ const invalidErr = new Error('foobar');
 isValidationErrorLike(err); // returns false
 ```
 
+### isZodErrorLike
+
+A [type guard](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) utility function, based on _heuristics_ comparison.
+
+_Why do we need heuristics since we can use a simple `instanceof` comparison?_ Because of multi-version inconsistencies. For instance, it's possible that a dependency is using an older `zod` version internally. In such case, the `instanceof` comparison will yield invalid results because module deduplication does not apply at npm/yarn level and the prototype is different.
```

Comment:
suggestion, nit: Since this is mentioned already above, I believe it can either be skipped or maybe better reference the previous instance, like:

We are using heuristics for error comparison. You can read why [here](https://github.com/causaly/zod-validation-error/tree/7b48d49d3db6bb23e9c907699282509aef2494f2?tab=readme-ov-file#isvalidationerrorlike).

Reply:
I actually did this deliberately, to avoid people from getting confused.